package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "0.17.5",
+  "version": "0.17.6",
```

```suggestion
  "version": "0.17.6-0",
```
Best to publish as a pre-release for testing